homeassistant/components/overkiz/button.py

```diff
@@ -65,6 +66,11 @@ class OverkizButtonDescription(ButtonEntityDescription):
         name="My position",
         icon="mdi:star",
     ),
+    OverkizButtonDescription(
+        key=OverkizCommand.CYCLE,
```

Comment:
Perhaps we should change the key to the OverkizCommand for all the others as well... Good find.

Reply:
Thx. Some commands are missing, I can do it in another PR once https://github.com/iMicknl/python-overkiz-api/pull/785 merged and released.